pwa/src/components/attributes/attribuutForm.tsx

```diff
@@ -19,6 +19,7 @@ import {
 } from "@conductionnl/nl-design-system/lib";
 import { isLoggedIn } from "../../services/auth";
 import FlashMessage from 'react-flash-message';
+import ElementCreationNew from '../common/elementCreationNew'
```

Comment:
Graag nog even hernoemen en eventueel de oude implementatie verwijderen voordat we 'm naar `dev` mergen.

Reply:
we gebruiken voor nu nog de oude implementatie. Dus haal de import weg en hernoem het als we de switch hebben gemaakt naar de nieuwe implementatie!